### PR TITLE
Disable batch/v2alpha1 API groupversion

### PR DESCRIPTION
**Pull request by liggitt**
We do not serve alpha APIs by default.

If required, they can be re-enabled with this config:

```
kubernetesMasterConfig:
  apiServerArguments:
    ...
    runtime-config:
    - apis/batch/v2alpha1=true
```

### CODE REVIEW

**Comment by soltysh** [comment]
> We do not serve alpha APIs by default.

Except for `batch/v2alpha1` because of cronjobs. But since `batch/v1beta1` is present since 3.9 this is ok.

File: vendor/k8s.io/kubernetes/pkg/kubectl/resource/builder.go [hunk 1]
```diff
@@ -219,6 +220,8 @@ func (b *Builder) LocalParam(local bool) *Builder {
 func (b *Builder) Local() *Builder {
 	mapper := *b.mapper
 	mapper.ClientMapper = DisabledClientForMapping{ClientMapper: mapper.ClientMapper}
+	// use the compiled in mappings, and tolerate unknown mappings
+	mapper.RESTMapper = NewRelaxedRESTMapper(legacyscheme.Registry.RESTMapper())
```
[the remark this answers is not in the document]
Comment (deads2k) [reply vendor/k8s.io/kubernetes/pkg/kubectl/resource/builder.go]: Yeah.  It should be fixable in 3.11

**Pull request merged**